Apps/WebClient/test/functional/e2e/cypress/db/seed.sql

```diff
@@ -0,0 +1,90 @@
+
+/* Unregistered HealthGateway User */
+DELETE FROM gateway."UserProfile" WHERE "UserProfileId" = 'S22BPV6WHS5TRLBL4XKGQDBVDUKLPIRSBGYSEJAHYMYRP22SP2TA';
```

Comment:
I think that we should run truncate on significant tables (userprofile, email, comms, etc) and then do the inserts.  This will  ensure that we have no side effects as the tables will be empty.

Reply:
right, unless we have cascade delete. Do we need a new task for producing a db diagram to make sure we don't miss any tables?

Reply:
I am not 100% doing sql inserts will work over time. Instead we should use the APIs to create said users and even delete them. That way we also test the business logic we have for that as well as make sure they are created the same way other users do

Reply:
I like the APIs approach which is implemented and recommended by big guys such as Microsoft in their CRM products. They have a rule saying "no direct access to SQL" :-)

Reply:
We shouldn't expand the scope of this ticket... It should take very little time to identify the tables we need truncated.  This ticket is to get the DB to a clean slate and have just enough data to run our tests.  We will be using the UI to create the vast majority of data but there are likely some situations where we need base data (validated SMS or email and I'm sure others).  The data inserted should come from a working application (locally).

Reply:
yeah, thats fine for now. In the future, we should make sure that we are running the full business logic otherwise we will have to continually improve this script